Library/.rubocop.yml

```diff
@@ -8,6 +8,19 @@ FormulaAudit:
 FormulaAuditStrict:
   Enabled: true
 
+# enable all pending rubocops
+AllCops:
+  NewCops: enable
+
+# don't allow cops to be disabled in formulae
+Style/DisableCopsWithinSourceCodeDirective:
+  Enabled: true
+  Exclude:
+  # TODO: really long lines but hard to resolve (but would be nice to do).
+  - '**/Formula/libgraphqlparser.rb'
+  # TODO: false positive in RuboCop, see if it can be worked around/fixed.
+  - '**/Formula/rpm.rb'
```

Comment:
Do you know is it reported to rubocop?

Reply:
It's not _really_ a bug on RuboCop's end because it is a formatting character it's just not being used as one in this context.

Reply:
May have found a fix: https://github.com/Homebrew/homebrew-core/pull/53261